sopel/modules/reddit.py

```diff
@@ -124,6 +124,7 @@ def redditor_info(bot, trigger, match=None):
     try:
         u = r.get_redditor(match.group(2))
     except Exception:  # TODO: Be specific
+        u = r.redditor(match.group(2))
```

Comment:
This won't do anything… This `except` block will always `return` no matter what, because of the `if`/`else` below it.